scripts/sync_protobuf/tensorflow.py

```diff
@@ -53,17 +57,20 @@
 XLA_IMPORT_PATTERN = re.compile(r"(\[|\s)xla\.")
 
 
+def move_tree(source: StrPath, destination: StrPath, /):
```

Comment:
Why not just use Path as the annotation? We use StrPath in stubs because we want to be permissive in accepting objects users might be passing, but for internal code like this it's better to enforce the type we generally want.

Reply:
Just out of habit, but sure this isn't library code so I can use Path instead to simplify the import.